packages/devtools_app/lib/src/screens/deep_link_validation/deep_links_screen.dart

```diff
@@ -238,10 +347,62 @@ class _ValidationDetailScreen extends StatelessWidget {
             ' and Custom Schemes in your app. Web check are done for the web association'
             ' file on your website. App checks are done for the intent filters in'
             ' the manifest and info.plist file, routing issues, URL format, etc.',
-            style: Theme.of(context).textTheme.bodySmall,
+            style: textTheme.bodyMedium!.copyWith(
+              color: colorScheme.onSurface.withOpacity(0.6),
+            ),
+          ),
+          if (tableView != TableViewType.pathView) ...[
+            const SizedBox(height: intermediateSpacing),
+            Text('Domain check', style: textTheme.titleSmall),
+            _DomainCheckTable(linkData: linkData),
+          ],
+          if (tableView != TableViewType.domainView) ...[
+            const SizedBox(height: intermediateSpacing),
+            Text('Path check (coming soon)', style: textTheme.titleSmall),
```

Comment:
is this still supposed to be tagged "coming soon"?

Reply:
Yes, this is part of the design. 
<img width="1540" alt="WeChat9adc5a2cf268e74e97d1aa28d3a746c3" src="https://github.com/flutter/devtools/assets/108393416/7843d385-041b-482c-9285-10d185c4a5c2">